cmd/clusterawsadm/credentials/credentials.go

```diff
@@ -0,0 +1,88 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package credentials
+
+import (
+	"bytes"
+	"encoding/base64"
+	"text/template"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/defaults"
+)
+
+// AWSCredentialsTemplate generates an AWS credentials file that can
+// be loaded by the various SDKs.
+const AWSCredentialsTemplate = `[default]
+aws_access_key_id = {{ .AccessKeyID }}
+aws_secret_access_key = {{ .SecretAccessKey }}
+region = {{ .Region }}
+{{if .SessionToken }}
+aws_session_token = {{ .SessionToken }}
+{{end}}
+`
+
+const AWSDefaultRegion = "us-west-2"
```

Comment:
any reason for west-2? seems like east-1 is typically the default...

Reply:
changing